pkg/apis/componentconfig/cloudcore/v1alpha1/helper.go

```diff
@@ -46,6 +48,18 @@ func (c *CloudCoreConfig) Parse(filename string) error {
 	return nil
 }
 
+func (c *CloudCoreConfig) GetNamespace() string {
+	if c.Namespace != "" {
+		return c.Namespace
+	}
+
+	if ns := os.Getenv(constants.NamespaceEnvKey); ns != "" {
+		return ns
+	}
+
+	return constants.SystemName
```

Comment:
We should override the configuration file fields with environment variables first

Reply:
Yeah

Reply:
> We should override the configuration file fields with environment variables first

I think we no need do that, 
```
if ns := os.Getenv(constants.NamespaceEnvKey); ns != "" {
	return ns
}
```
can be removed.

Because the external iptables can directly get env key and call SystemNamespaceInit:
```
constants.SystemNamespaceInit(config.GetNamespace())
```

What do you think?

Reply:
For cloudcore, we will read from cloudcore.yaml. For external iptablesmgr, we will read from env key NAMESPACE to get the given namespace.

Reply:
cloudcore is also no need to add new config field for namespace I think.. @zhu733756 @zc2638